src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2021,6 +2023,29 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     {
         if (app_.config().VALIDATION_PUB.size ())
         {
+            auto const validation_manifest =
+                app_.config().section (SECTION_VALIDATION_MANIFEST);
+
+            if (! validation_manifest.lines().empty())
+            {
+                std::string s;
```

Comment:
`s.reserve` here. I think 188 is the right value (though if it's a pain to confirm, I'm OK using the next "nice" number after that: 200 or 256).